sites/foresternetwork/config/native-x.js

```diff
@@ -0,0 +1,41 @@
+module.exports = {
+  enabled: true,
+  uri: 'https://ebm.native-x.io',
+  placements: {
+    default: {
+      card: { id: '5d372175f1658300018c023e' },
+      list2: { id: '5d372175f1658300018c023e' },
+      list1: { id: '5d372175f1658300018c023e' },
+    },
```

Comment:
Yes, I used the same card placement for all of these.  Thought is to unify/rename them post-launch since we only need one these days.